go/dynamodb/ScanItems/ScanItems.go

```diff
@@ -0,0 +1,131 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[dynamodb.go.scan_table_items]
+package main
+
+// snippet-start:[dynamodb.go.scan_table_items.imports]
+import (
+    "flag"
+    "fmt"
+    "strconv"
+
+    "github.com/aws/aws-sdk-go/aws/session"
+    "github.com/aws/aws-sdk-go/service/dynamodb"
+    "github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+    "github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+// snippet-end:[dynamodb.go.scan_table_items.imports]
+
+// Item holds info about new item
+// snippet-start:[dynamodb.go.scan_table_items.struct]
+type Item struct {
+    Year   int
+    Title  string
+    Plot   string
+    Rating float64
+}
+
+// snippet-end:[dynamodb.go.scan_table_items.struct]
+
+// ScanTableItems retrieves the table items released in a specific year
+// Inputs:
+//     svc is a LambdaDB service client
+//     table is the name of the table
+//     year is when the movie was released
+// Output:
+//     If success, the results of the scan and nil
+//     Otherwise, nil and an error from the call to Scan
+func ScanTableItems(sess *session.Session, year *int, table *string, minRating *float64) (*[]Item, error) {
+    // snippet-start:[dynamodb.go.scan_table_items.expr]
+    // Create the Expression to fill the input struct with.
```

Comment:
...Create the **expression to fill the input struct** ~~with.~~

Reply:
Fixed